ConcurrencyHomework/src/concurrency/Main.java

```diff
@@ -0,0 +1,64 @@
+package concurrency;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+
+        List<String> personInfo = new ArrayList<>();
+
+        //read information from the file 1
+        try (BufferedReader file1 = new BufferedReader(new InputStreamReader(new FileInputStream("D:\\JAVA EX\\scoala-informala-homework\\ConcurrencyHomework\\file1.txt")))) {
```

Comment:
Logica de citire a fisierului ar fi trebuit grupata cu cea de creare a obiectelor `Person` si ar fi trebuit extrase in alta clasa/metoda. astfel incat sa poti apela acea metoda din thread-uri diferite.